contracts/Vault.sol

```diff
@@ -30,19 +27,31 @@ contract Vault is
     constructor(
         address _underlying,
         uint256 baseUnit,
-        address[] memory _strategies
+        address[] memory _strategies,
```

Comment:
Choosing not to include strats in constructor might allow us to have vault deployed without a strategy initially

Reply:
Had a think on this. The current setup means the vault doesn't accept deposits unless there is a strategy.

Think for now we should leave it - a vault without a strategy doesn't make a whole lot of sense.